docs/src/modules/components/SkipLink.tsx

```diff
@@ -6,7 +6,7 @@ import { useTranslate } from 'docs/src/modules/utils/i18n';
 const StyledLink = styled(MuiLink)(({ theme }) => ({
   position: 'fixed',
   padding: theme.spacing(1),
-  background: theme.palette.background.paper,
+  background: (theme.vars || theme).palette.background.paper,
```

Comment:
This creates a hydration mismatch otherwise when the page is in dark mode.

<img width="437" alt="Screenshot 2023-01-28 at 01 53 15" src="https://user-images.githubusercontent.com/3165635/215232226-e52040a7-b6a7-4704-8a3f-e141119323bc.png">

http://0.0.0.0:3000/blog/v6-beta-pickers/